cli_docs/generate_cli_docs.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import sys
+
+import click
+from pkg_resources import iter_entry_points
+
+
+def generate_docs(name, filename):
+    console_scripts = [ep for ep in iter_entry_points('console_scripts', name=name)]
+
+    if len(console_scripts) == 0:
+        raise click.ClickException('"{}" is not an installed console script.'.format(name))
+
+    # read all lines from original file
+    original_lines = []
+    with open(filename, "r") as md_file:
+        original_lines = md_file.readlines()
+
+    with open(filename, "w") as md_file:
+        # copy header from original file
+        for line in original_lines:
+            md_file.write(line)
+            if "AUTOGEN_START" in line:
+                break
```

Comment:
Right now it is:
1. Load packages
2. Generate header
3. generate data from packages.

I think that it makes sense not to split this flow, and do:
1. Generate header
2. Load packages
3. Generate data from packages